aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/ServletLambdaContainerHandlerBuilder.java

```diff
@@ -178,6 +178,7 @@ public Builder asyncInit(long actualStartTime) {
     /**
      * Uses a new {@link AsyncInitializationWrapper} with the no-parameter constructor that takes the actual JVM
      * start time
+     * @deprecated As of release 2.1 this method is deprecated. Initializer is always async if running in on-demand.
```

Comment:
should be 2.0.0 (currently we just have 2.0.0-M1 and 2.0.0-M2 milestone releases out there)